### DISCUSSION TITLE
Does -s flag keeps Adobe meta (for correct file viewing)?

### DISCUSSION THREAD

**Discussion by necros2k7** [ImageProcessing-ElectronicPublications/jpeg-recompress#8] [Q&A]
@zvezdochiot @xiota

**Comment by xiota** [comment]
I don't use that option, but would consider it a bug if it doesn't strip all metadata.

https://github.com/ImageProcessing-ElectronicPublications/jpeg-recompress/blob/300d18bf17e7f76aeae3ddf02dfa0b678102bb99/src/jpeg-recompress.c#L278-L281

https://github.com/ImageProcessing-ElectronicPublications/jpeg-recompress/blob/300d18bf17e7f76aeae3ddf02dfa0b678102bb99/src/jpeg-recompress.c#L432-L434

**Comment by zvezdochiot** [comment]
What does Adobe have to do with this?
